src/versions.js

```diff
@@ -0,0 +1,4 @@
+export const siteVariables = {
+    dipVersion: 'Feature available in version 1.12.0 of the KILT runtime and higher',
```

Comment:
Isn't DIP only available on the 1.12.0 version of peregrine, but not yet on spiritnet?